.github/workflows/ci_compliance.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: Compliance
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  duvet:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: goatgoose/duvet-action@v1.0.0
```

Comment:
Should we be using an action in your private repo?

Reply:
Moved to the action in s2n-quic in 447bc32